test/sqlsrv/srv_007_login_timeout.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+False connection with LoginTimeout option
+--DESCRIPTION--
+Intentionally provide an invalid server name and set LoginTimeout. Verify the time elapsed.
+--SKIPIF--
+<?php require('skipif.inc'); ?>
+--FILE--
+<?php
+
+$serverName = "WRONG_SERVER_NAME";
+
+$t0 = round(microtime(true));
+
+$conn = sqlsrv_connect( $serverName , array("LoginTimeout" => 8));
+
+$t1 = round(microtime(true));
+
+echo "Connection attempt time: " . ($t1 - $t0) . " [sec]\n";
+
+print "Done";
+?>
+
+--EXPECTREGEX--
+Connection attempt time: [7-9] \[sec\]
```

Comment:
Since it will definitely fail, it should wait at most 8 seconds, but the way we measure the time is not very accurate. I've seen 9 seconds a few times (though not 7 seconds).